search-services/latest/using/index.md

```diff
@@ -358,12 +358,11 @@ These examples show how to embed queries in CMIS.
 
 ```sql
 - strict queries
-SELECT * FROM Document WHERE CONTAINS("zebra")
```

Comment:
Hi @abhinavmishra14, I forgot to ask why this line was removed? This was an oversight on my part!